src/components/data-management/ProjectMenu.jsx

```diff
@@ -174,20 +28,14 @@ const ProjectMenu = () => {
           Add samples
         </Button>
         <DownloadData />
```

Comment:
small thing: you can call this `DownloadDataButton` so that it is exactly like `LaunchAnalysisButton`.

Reply:
changed